src/site/layouts/landing_page.drupal.liquid

```diff
@@ -0,0 +1,9 @@
+{% include "src/site/includes/header.html" %}
+<main>
+  <div class="row">
+    <h1>{{ title }}</h1>
+    <div class="usa-font-lead">{{ fieldIntroText }}</div>
+  </div>
+</main>
+{% include "src/site/includes/footer.html" %}
+{% include "src/site/includes/debug.drupal.liquid" %}
```

Comment:
Right now, we use the concept of layouts and templates. A "template" is sort of a sub-layout. This doesn't really make sense to me, because our most widely-used layout, `page-breadcrumbs.html`, is largely just a switch statement between templates, so it's just added complexity. 

I think we can use the Drupal integration as an opportunity to clean this up.

Reply:
Yeah, that division has never made sense to me either.